pkg/controller/persistentvolume/persistentvolume_claim_binder_controller_test.go

```diff
@@ -186,11 +186,13 @@ func TestBindingWithExamples(t *testing.T) {
 	if err != nil {
 		t.Errorf("Unexpected error getting PV from client: %v", err)
 	}
+	pv.ObjectMeta.SelfLink = testapi.Default.SelfLink("pv", "")
```

Comment:
I think the ```name``` should be ```any```. as that is the name being requested on [line 184](https://github.com/aveshagarwal/kubernetes/blob/master-selflink-objref/pkg/controller/persistentvolume/persistentvolume_claim_binder_controller_test.go#L184). Using (or not using) the name doesn't seem to break anything in testing but it reads easier.

Similar notes for other ```testapi.Default.SelfLink``` calls.